DOCS-2160: How to add website/customized logo to Fastcheckout (#1220)
MultiSafepay/docs: content/payment-methods/fastcheckout/what-is-fastcheckout.md

__Not available__: Klarna, Pay After Delivery. 

### Adding a website and customized logo

> To add a website, please refer to our documentation on how to [add your website](/tools/multisafepay-control/add-website/)

Reviewer: ```suggestion
For instructions on how to add a website, see MultiSafepay Control - [add your website](/tools/multisafepay-control/add-website/).
```

> To add your personal logo to the Fastcheckout payment pages, please follow these steps:

Reviewer: ```suggestion
To add your personal logo to the Fastcheckout payment pages, follow these steps:
```

> 1. Log in to your [MultiSafepay Control](https://merchant.multisafepay.com/)

Reviewer: ```suggestion
1. Log in to your [MultiSafepay Control](https://merchant.multisafepay.com/).
```

> 2. Navigate to _Settings_ → _Website settings_

Reviewer: ```suggestion
2. Navigate to **Settings** → **Website settings**.
```

> 3. Scroll down to the option _Payment logo (Fast Checkout)_

Reviewer: ```suggestion
3. Scroll down to the option **Payment logo (Fast Checkout)**.
```

4. Select your logo

> 5. Click _Save_

Reviewer: ```suggestion
5. Click **Save**.
```
